README.md

```diff
@@ -21,6 +21,18 @@ Inside the project, install the project's dependencies:
 cd igbo_api/
 yarn install
 ```
+If you are running the API on windows Operating system, run the module to install it globally:
```

Comment:
small detail: 'Windows' should be uppercase and 'operating' should be lowercase.

Reply:
thats cool I will fix it right away